### ISSUE TITLE
Is there any suggest for develope a GantChart for F-Visualization?

### ISSUE THREAD

**Issue by saman222** [joomlahenk/fabrik#422]
Hi
i need use GantChart with Fabrik. is there any JS or PHP lib to implement that as a Fabrik Visualization ? i want spend time to develop that part of fabrik4. it seems old fusion_gantt_chart which used in fabrik3, now is not free. is there any other suggestion?

**Comment by skurvish** [comment]
You are welcome to give it a try if you like. One option that I have looked at is integrating KoolReports as a library to Fabrik. Koolreports has lots of visualization options, Gant Chart being one of them. You might look at what would be required to implement such a solution.

**Comment by saman222** [comment]
KoolReport has some commercial EXTENDED PACKAGES. are you sure it is good and enough? what is your idea about other lib? i can't find suitable lib.

**Comment by skurvish** [comment]
It is unlikely you will get any of us onboard to do any work on this. We are way too busy just getting the stuff we know we need working. Maybe some day. But if you want to take a stab at it feel free. We can answer questions if you have any.

**Comment by saman222** [comment]
> It is unlikely you will get any of us onboard to do any work on this. We are way too busy just getting the stuff we know we need working. Maybe some day. But if you want to take a stab at it feel free. We can answer questions if you have any.

are you confirm to use JS lib?
https://www.highcharts.com/blog/download/
it can be used free (but has watermark).
please see demos:
https://www.highcharts.com/demo/gantt

**Comment by raramuridesign** [comment]
@saman222 I would suggest you look at chartjs.org and the plugin for gantt
https://github.com/anton-shchyrov/chartjs-plugin-gantt
https://www.npmjs.com/package/chartjs-plugin-gantt

Since this library is opensource. Even though highcharts is a very good tool, users may not like the watermark and usage may be restricted in the future. Not the case with chartjs

Hope this helps.

**Comment by saman222** [comment]
> @saman222 I would suggest you look at chartjs.org and the plugin for gantt https://github.com/anton-shchyrov/chartjs-plugin-gantt https://www.npmjs.com/package/chartjs-plugin-gantt
> 
> Since this library is opensource. Even though highcharts is a very good tool, users may not like the watermark and usage may be restricted in the future. Not the case with chartjs
> 
> Hope this helps.

yes i agree that is opensource and is better in this case, but it seems weak in GantChart with plugin witch you get link.
https://www.fusioncharts.com/javascript-charting-comparison/highcharts-vs-chartjs
